### ISSUE TITLE
Update `clusteradm install hub-addon` to wrap Helm chart repo

### ISSUE THREAD

**Issue by dhaiducek** [open-cluster-management-io/ocm#40]
Currently `clusteradm install hub-addon` uses YAML manifests hosted in the `clusteradm` repo to deploy hub addon components. This is undesirable for maintainability. With the new https://github.com/open-cluster-management-io/helm-charts repo, the hub-addon command should be updated to deploy helm charts from this repo directly.

Related issue:
- https://github.com/open-cluster-management-io/community/issues/136

**Comment by qiujian16** [comment]
@ycyaoxdu would it be possible to do this if we finish the standalone controlplane part?

**Comment by ycyaoxdu** [comment]
> @ycyaoxdu would it be possible to do this if we finish the standalone controlplane part?

@qiujian16 Yes, we used helm API to install the standalone controlplane. I think we can also use helm install API in other commands after that part is merged.

**Comment by qiujian16** [comment]
@mikeshng @dhaiducek i am wondering if you are ok to build a helm chart for the addon and put into the helm chart repo (we have an action workflow to automate it already). So that we will be able change this in the clusteradm.

**Comment by mikeshng** [comment]
I am ok with that. As part of that work, I am going to change the current app addon to the Argo CD pull model one.

**Comment by dhaiducek** [comment]
We're good with that for GRC also. I guess this would target the next release?

**Comment by mikeshng** [comment]
The new Helm chart wrapped app-addon has been merged.

**Comment by mikeshng** [comment]
@dhaiducek you should have access in this repo to manually remove it.

**Comment by dhaiducek** [comment]
> [@dhaiducek](https://github.com/dhaiducek) you should have access in this repo to manually remove it.

I do not.

**Comment by mikeshng** [comment]
My bad. I thought this is the clusteradm repo based on the name of the issue.
